### PR TITLE
NISQ Analyzer Results in QHAna

### PR DESCRIPTION
**Pull request by cloudsftp**

### CODE REVIEW

File: src/app/components/experiment-workspace/experiment-workspace.component.ts [hunk 1]
```diff
@@ -207,7 +212,7 @@ export class ExperimentWorkspaceComponent implements OnInit, OnDestroy {
             }
         }
         this.activePlugin = plugin;
-        this.frontendUrl = frontendUrl;
+        this.frontendUrl = frontendUrl + `?plugin-endpoint-url=${plugin.url}`;
```
Comment (buehlefs) [inline src/app/components/experiment-workspace/experiment-workspace.component.ts:215]: If the frontend URL should include this param, then the plugin can just include the param in the entry point uiHref. There is no need to change the QHAna UI for this!

File: src/app/components/plugin-uiframe/plugin-uiframe.component.ts [hunk 2]
```diff
@@ -392,7 +392,7 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
             if (data?.type === "form-submit") {
                 if (!isFormSubmitData(data)) {
                     return;
-                }
+                }https://angular.io/guide/hierarchical-dependency-injection#resolution-modifiers
```
Comment (buehlefs) [inline src/app/components/plugin-uiframe/plugin-uiframe.component.ts:395]: ??

File: src/app/components/plugin-uiframe/plugin-uiframe.component.ts [hunk 3]
```diff
@@ -425,6 +425,22 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
                 }
                 this.handlePluginInfoRequest(data);
             }
+            if (data.type === "nisq-analyzer-result") {
```
Comment (buehlefs) [inline src/app/components/plugin-uiframe/plugin-uiframe.component.ts:428]: The nisq analyzer can and should use the form-submit message! There is no nead for this special message type here.

File: src/app/components/plugin-uiframe/plugin-uiframe.component.ts [hunk 4]
```diff
@@ -297,14 +296,18 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
     }
 
     private loadImplementations(): void {
+        const experimentId = this.parent.experimentId;
```
Comment (buehlefs) [inline src/app/components/plugin-uiframe/plugin-uiframe.component.ts:299]: These changes (making the workspace injectable and not using the currentExperimentServce to get the experimentId) seem unrelated. Why are these changes required?

**Review by buehlefs — CHANGES_REQUESTED** [review]
In theory, there is no need to change the QHAna UI for this feature of the NISQ Analyzer plugin (see comments).

**Comment by cloudsftp** [comment]
Implemeted without changes to QHAna UI
